doc/sly.texi

```diff
@@ -774,7 +774,8 @@ a @code{.lisp} file.
 These commands each evaluate a Common Lisp expression in a different
 way.  Usually they mimic commands for evaluating Emacs Lisp code.  By
 default they show their results in the echo area, but a prefix
-argument causes the results to be inserted in the current buffer.
+argument causes the results to be inserted in the current buffer, and
```

Comment:
Minor nit: maybe "while a negative..." would read better then "and a negative". Personal preference.